main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+)
+
+func main() {
+	const chunkSize = 4096
+	cArgs := os.Args[1:]
+	buf := bufio.NewReader(os.Stdin)
+	buffer := make([]byte, chunkSize)
+
+	for {
+		_, dataErr := buf.Read(buffer)
```

Comment:
Why _?

Reply:
because buf.Read return the number of bytes read into buffer and the actual data is located in buffer

Reply:
Yes, "buf.Read return the number of bytes read into buffer". And if your input is less than 4096 bytes?